_posts/2016-05-07-sensitivity-analysis.md

```diff
@@ -1,6 +1,85 @@
 ---
 layout: post
 title: Sensitivity Analysis
-img: image-5.png
+img: sensitivity.png
 ---
 
+Sensitivity analysis helps to the study of how the variations in the outputs of a numerical or 
+physical system can be assigned to different sources of variation in its inputs.
+
+Understanding the variance of the outputs of a system in terms of the contributions of the inputs is a key aspect of
+uncertainty quantification: it helps to test the robustness of a system, design strategies for uncertainty reduction
+and helps to calibrate physical models to the real world by improving their interpretability.
+
+Sensitivity analysis methods are divided in two main classes: *local* methods, where changes in the output are studied for
+specific values of the inputs, and *global* methods where the total variability of the outputs is assigned to each input variable. Given a physical model of a system, for instance a climate model simulator, local sensitivity analysis is carried out using gradients.
+However, if the physical model is expensive-to-evaluate approximating gradients becomes intractable. Using an emulator of the physical model, like
```

Comment:
```suggestion
However, if the physical model is expensive-to-evaluate approximating gradients become intractable. Using an emulator of the physical model, like
```

Reply:
I think Javier's version was correct. The become refers to the word 'approximating'.